app/src/main/java/com/vero/tasky/auth/domain/usecase/ValidatePasswordUseCase.kt

```
@@ -0,0 +1,13 @@
+package com.vero.tasky.auth.domain.usecase
+
+import java.util.regex.Pattern
+
+class ValidatePasswordUseCase {
+
+    private val pattern = Pattern.compile("^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)")
+
+    operator fun invoke(password: String): Boolean {
+        return password.length > 8 && pattern.matcher(password).matches()
```

Here you could return an error if password is less than 8
And another error if password is > 8 but doesnt include a digit, lowercase or uppercase

So the user is aware of what they need to fix in their password